Gemfile.lock

```diff
@@ -1,94 +1,94 @@
+    actionmailer (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      actionview (= 6.0.3.4)
+      activejob (= 6.0.3.4)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.4)
+      actionview (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actiontext (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       nokogiri (>= 1.8.5)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.4)
+      activesupport (= 6.0.3.4)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    activejob (6.0.3.4)
+      activesupport (= 6.0.3.4)
       globalid (>= 0.3.6)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activestorage (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
+    activemodel (6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activerecord (6.0.3.4)
+      activemodel (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activestorage (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
       marcel (~> 0.3.1)
-    activesupport (6.0.3.2)
+    activesupport (6.0.3.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     aws-eventstream (1.1.0)
-    aws-partitions (1.353.0)
-    aws-sdk-core (3.104.3)
+    aws-partitions (1.399.0)
+    aws-sdk-core (3.109.3)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.36.0)
-      aws-sdk-core (~> 3, >= 3.99.0)
+    aws-sdk-kms (1.39.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.76.0)
-      aws-sdk-core (~> 3, >= 3.104.1)
+    aws-sdk-s3 (1.85.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.2.1)
+    aws-sigv4 (1.2.2)
       aws-eventstream (~> 1, >= 1.0.2)
-    bcrypt (3.1.15)
-    better_errors (2.7.1)
+    bcrypt (3.1.16)
+    better_errors (2.9.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
     bindex (0.8.1)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.7)
+    bootsnap (1.5.1)
```

Comment:
### [bootsnap](https://github.com/Shopify/bootsnap)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.4.7 → 1.5.1

[_[change-log](https://github.com/Shopify/bootsnap/blob/master/CHANGELOG.md), [source-code](https://github.com/Shopify/bootsnap)_]





<details>
<summary>Commits</summary>

A change of **30** commits. See the full changes on [the compare page](https://github.com/Shopify/bootsnap/compare/v1.4.7...v1.5.1).

These are the first 10 commits:
- (29b77ee) [Prevent Bootsnap::LoadPathCache::FallbackScan from polluting…](https://github.com/Shopify/bootsnap/commit/29b77ee23de14a8a1f907ee65f833d6b0a1815b2)
- (280ff11) [Remove unnecessary files from gem](https://github.com/Shopify/bootsnap/commit/280ff1173c2a138f162d0ef422098529760e674b)
- (d7f33a6) [Merge pull request #315 from t-richards/chore/gem-size](https://github.com/Shopify/bootsnap/commit/d7f33a60d0cf78bb1f17947caf2e202be5b8772b)
- (b1096d1) [Merge pull request #314 from aaronjensen/fix-error-cause-pol…](https://github.com/Shopify/bootsnap/commit/b1096d1d13704c3972216e73c6479ef040144d9f)
- (e8aad0c) [bump version to 1.4.8](https://github.com/Shopify/bootsnap/commit/e8aad0ce200e05c935e0952aadc2e8a5e95e13f4)
- (cfe5b4b) [CompileCache::YAML add support for symbolize_names and freez…](https://github.com/Shopify/bootsnap/commit/cfe5b4b195aa8363b904c72c25543ce66afe6873)
- (737ff88) [Merge pull request #320 from Shopify/yaml-cache-arguments](https://github.com/Shopify/bootsnap/commit/737ff88c3c43a715cf9d58e60ead5d716737f118)
- (fae33c3) [Enable Windows support](https://github.com/Shopify/bootsnap/commit/fae33c350baa32471fee6db6ade64079a393dec2)
- (26e4b6f) [Enable testing using Github Actions](https://github.com/Shopify/bootsnap/commit/26e4b6ff3a671809abcf26f849cb944dac18bfe2)
- (0cc8fbf) [Fix tests for windows platform](https://github.com/Shopify/bootsnap/commit/0cc8fbf3e9fc834c7e968fd9f7abe00bb2f50172)


</details>